app/views/layouts/application.html.erb

```diff
@@ -18,13 +18,11 @@
         <%= render 'layouts/navigation' %>
       </header>
     <% end %>
-    <main role="main">
-      <div class="container">
-        <%= render 'layouts/messages' %>
+    <main class="container" role="main">
```

Comment:
`main` it turns out extended container, so we take it out of the css and just give it the class to make it explicit (and not double apply)